cPP_DSC.adoc

```diff
@@ -443,15 +443,15 @@ The PPs and PP-Modules that are allowed to be specified in a PP-Configuration wi
 
 *T.BRUTE_FORCE_AUTH:* An unauthorized user may attempt to gain unauthorized access to the TOE by repeatedly and rapidly supplying a large number of permutations of authorization data, such as passwords, biometrics, etc. that protect the SDEs, in the hopes that valid authorization data can be obtained through brute force.
 
-*T.HW_ATTACK:* An individual with physical access to the TOE may apply hardware attacks such as probing, physical manipulation, fault injection, environmental stress, or reactivating blocked test-features or other pre-delivery services to manipulate the behavior of the TOE to disclose SDOs.
+*T.HW_ATTACK:* An individual with physical access to the TOE may apply hardware attacks such as probing, physical manipulation, fault injection, environmental stress, or reactivating blocked test-features or other pre-delivery services to manipulate the behavior of the TOE to disclose SDEs.
```

Comment:
OK, I can agree with this change, but it seems that it leads to an update that should be made to T.UNAUTHORIZED_ACCESS that would remove the "SDOs" from SDEs/SDOs. This would then make all the threats consistent with claims to SDEs.

Reply:
@woodbe I made that change.